scene/animation/animation_blend_tree.cpp

```diff
@@ -728,7 +763,7 @@ float AnimationNodeTransition::get_cross_fade_time() const {
 }
 
 double AnimationNodeTransition::process(double p_time, bool p_seek, bool p_seek_root) {
-	int current = get_parameter(this->current);
+	int current = tree_root->get_parameter(String(node_name) + "/" + this->current);
```

Comment:
We should not be doing unnecessary string ops in `process`. This should be cached. (Same for all nodes which do this)

Reply:
I agree, if the rest of the PR is approved then I'll come back to that point and add the cache.